apps/services/user-notification/src/app/modules/notifications/notifications.service.ts

```diff
@@ -285,4 +286,30 @@ export class NotificationsService {
       )
     }
   }
+
+  @Cron('0 * * * *') // run once every hour
```

Comment:
Is it necessary to do this every hour?
If the resolution is for six months is it not sufficient f.x. to do it once a day with f.x: `@Cron('0 0 * * *')`

Reply:
true - I just want to see this working on dev hourly to begin with, and then followup PR changing the cron a once a day

Reply:
Hey, just noticed this while browsing.

How about using '@hourly' or '@weekly'?

Reply:
splendid idea